docs/source/getting-started/installing.adoc

```diff
@@ -45,6 +45,10 @@ https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/e
 https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[add a Virtual Switch]
 using the Hyper-V Manager. Make sure that you pair the virtual switch
 with a __network card (wired or wireless) that is connected to the network__.
+In case, you have any existing Virtual Switch like _DockerNAT_ which comes when installing https://docs.docker.com/docker-for-windows/install[docker for Windows],
+then you need to add the name of newly created virtual switch into environment variable
+`HYPERV_VIRTUAL_SWITCH` as `$env:HYPERV_VIRTUAL_SWITCH=name_of_switch`.
+You can find more info in link:../using/managing-minishift{outfilesuffix}[Managing Minishift] page under _Driver specific environment variables_ section.
```

Comment:
the link should go directly to the appropriate section. I guess that won't work, until the other pull request is merged.